haystack/utils/augment_squad.py

```diff
@@ -50,14 +48,23 @@ def load_glove(
     glove_path: Path = Path("glove.txt"),
     vocab_size: int = 100_000,
     device: Union[str, torch.device] = torch.device("cpu:0"),
+    timeout: Union[float, Tuple[float, float]] = 10.0,
 ) -> Tuple[dict, dict, torch.Tensor]:
-    """Loads the GloVe vectors and returns a mapping from words to their GloVe vector indices and the other way around."""
+    """
+    Loads the GloVe vectors and returns a mapping from words to their GloVe vector indices and the other way around.
+    :param timeout: How many seconds to wait for the server to send data before giving up,
+        as a float, or a :ref:`(connect timeout, read timeout) <timeouts>` tuple.
+        Defaults to 10 seconds.
+    :type timeout: Union[float, Tuple[float, float]]
```

Comment:
No need to add the type here as we already have it in the type annotation in the methods signature